app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
   has_many :comments, foreign_key: :author_id
   has_many :posts, foreign_key: :author_id
   has_many :likes, foreign_key: :author_id
 
-  validates :name, presence: true
-  validates :post_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :name, presence: false
+  # validates :post_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Try not to remove the validation, as it helps to save the actual data while creating the attributes. Kindly put the validation and fill out the form & check it is submitted successfully.

-> * If validation is failing it means the parameter in `User.create` is not correct, Check whether getting the correct data.